app/routes/error/route.tsx

```diff
@@ -1,11 +1,20 @@
-import { useSearchParams } from '@remix-run/react';
+import { useNavigate, useSearchParams } from '@remix-run/react';
 import { ErrorComponent } from '~/components/error-component/error-component';
+import { ROUTES } from '~/router/config';
 
 export default function ErrorPage() {
     const [searchParams] = useSearchParams();
+    const navigate = useNavigate();
 
-    const title = searchParams.get('title');
-    const message = searchParams.get('message');
+    const title = searchParams.get('title') ?? 'Ooops, something went wrong';
+    const message = searchParams.get('message') ?? undefined;
```

Comment:
what is a `message` search param ?

Reply:
error page takes title and message from the search params